Docs/source/running_cpp/parameters.rst

```diff
@@ -310,14 +310,12 @@ Particle initialization
     The mass of one `physical` particle of this species.
     If ``species_type`` is specified, the mass will be set to the physical value and ``mass`` is optional.
 
-* ``<species_name>.xmin,ymin,zmin`` (`float`) optional (default unlimited)
-    When ``<species_name>.xmin`` and ``<species_name>.xmax`` (see below) are set, they delimit the region within which particles are injected.
+* ``<species_name>.xmin,ymin,zmin`` and ``<species_name>.xmax,ymax,zmax`` (`float`) optional (default unlimited)
```

Comment:
I guess this might look better than it currently does (although I haven't actually compiled the doc to be honest)